bindings/python/libmata/nfa/strings.pxd

```diff
@@ -0,0 +1,38 @@
+from libcpp cimport bool
+from libcpp.set cimport set as cset
+from libcpp.string cimport string
+from libc.stdint cimport uintptr_t
+from libcpp.unordered_map cimport unordered_map as umap
+from libcpp.memory cimport shared_ptr
+from libcpp.vector cimport vector
+
+from libmata.nfa.nfa cimport CNfa, CTrans
+
+cdef extern from "mata/nfa.hh" namespace "Mata::Nfa":
+    ctypedef vector[CTrans] TransitionSequence
+    ctypedef vector[CNfa] AutSequence
+    ctypedef vector[CNfa*] AutPtrSequence
+    ctypedef vector[const CNfa*] ConstAutPtrSequence
+    ctypedef umap[string, string] StringMap
```

Comment:
Just a note: Moving these types here is actually a good idea. We might want to replicate the same in C++ implementation.

Reply:
Yeah, I mainly tried to restrict the needed imports to modules, where they are used, but it seems to me these should be moved in upstream as well.